### ISSUE TITLE
License for this package?

### ISSUE THREAD

**Issue by wking** [spdx/license-test-files#5]
To be extra clear about the terms under which the contents are distributed, I suggest we land something in the README explaining that any license source and headers retain their original copyright terms, while anything unique to this repository, where copyrightable at all, is under the CC0-1.0.  Or something like that.

**Comment by jmanbeck** [comment]
Agreed. We are still debating that but I like CC-1.0. My caveat is that some of the license text generated includes copyrights because its in the official license text.... Wasn't sure how to handle that.  Wanted to ask the Legal team.

**Comment by wking** [comment]
> My caveat is that some of the license text generated includes copyrights because its in the official license text…

Right.  And those copyright lines are sometimes template examples (e.g. [here][1]) and sometimes actual claims for the license/header text (e.g. [here][2]).  I think your current:

> Any copyrights appearing in this test file do so because they were part of the license text as stored by SPDX and are included only for test purposes as they are part of the license text. They have no meaning, implied or specific, otherwise.                                                         

is an attempt to address that.

With this proposal, I'd like to see it adjusted to something like:

> This test file is an example from the license-test corpus and is included only for test purposes.  It includes the text of the upstream license or header, which may include template copyright claims, as well as actual copyright claims by the license authors.  The upstream license and header text in this file retains those claims, and their presence in this file does not change their copyright or licensing.
>
> Any content beyond the upstream license and header which occurs in this file was written in 2017 by license-test-files contributors.
>
> To the extent possible under law, the license-test-files contributors have dedicated all copyright and related and neighboring rights to this content to the public domain worldwide. This content is distributed without any warranty.
>
> You should have received a copy of the CC0 Public Domain Dedication along with this file. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.

which uses wording inspired by the [CC's recommendation][3].

And yeah, feedback from the legal team would be good.

[1]: https://github.com/spdx/license-list-XML/blob/f1522b5cc61bde64d9b38af05204fdb93c02eef8/src/BSD-2-Clause.xml#L7-L8
[2]: https://github.com/spdx/license-list-XML/blob/f1522b5cc61bde64d9b38af05204fdb93c02eef8/src/GPL-3.0.xml#L22
[3]: https://wiki.creativecommons.org/wiki/CC0_FAQ#May_I_apply_CC0_to_computer_software.3F_If_so.2C_is_there_a_recommended_implementation.3F
